persistent-mongoDB/persistent-mongoDB.cabal

```
@@ -1,5 +1,5 @@
 name:            persistent-mongoDB
-version:         2.9.0.2
+version:         2.9.0.3
```

Because we are changing the meaning of a function, it's going to have to be a major version bump.

```suggestion
version:         2.10.0.0
```